skimage/filters/thresholding.py

```diff
@@ -578,57 +578,43 @@ def threshold_minimum(image, nbins=256, bias='min', max_iter=10000):
     >>> binary = image > thresh
     """
 
-    def find_local_maxima(hist):
+    def find_local_maxima_idx(hist):
```

Comment:
I think plateaus, while unlikely, can still happen.  So we can't get rid of this, but in testing it I also find corner cases in which it fails.